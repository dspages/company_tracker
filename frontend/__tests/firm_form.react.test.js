


import React from 'react';
import FirmForm from '../components/firm_index/firm_form.jsx';
import renderer from 'react-test-renderer';


test('Component correctly renders', () => {
  const component = renderer.create(<FirmForm
    />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
