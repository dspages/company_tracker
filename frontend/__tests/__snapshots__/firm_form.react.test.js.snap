// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component correctly renders 1`] = `
<form
  className="new-form"
>
  <div
    className="new-form-row"
  >
    Firm Form
    <br />
    <label>
      Name:
      <input
        id="name"
        onChange={[Function]}
        type="text"
        value={undefined}
      />
    </label>
    <label>
      Info:
      <textarea
        id="info"
        onChange={[Function]}
        type="textarea"
        value={undefined}
      />
    </label>
    <label>
      Status:
      <select
        onChange={[Function]}
        value={undefined}
      >
        <option
          value="researching"
        >
          Researching
        </option>
        <option
          value="pending"
        >
          Pending
        </option>
        <option
          value="approved"
        >
          Approved
        </option>
        <option
          value="denied"
        >
          Denied
        </option>
      </select>
    </label>
    <button
      onClick={[Function]}
    >
      Confirm
    </button>
  </div>
</form>
`;
